resources/firewalld_policy.rb

```diff
@@ -0,0 +1,96 @@
+unified_mode true
+
+provides :firewalld_policy,
+         os: 'linux'
+
+property :description,   # (s): see description tag in firewalld.policy(5).
+         String
+property :egress_zones,  # as: array of zone names. See egress-zone tag in firewalld.policy(5).
+         [Array, String]
+property :forward_ports, # (a(ssss)): array of (port, protocol, to-port, to-addr). See forward-port tag in firewalld.policy(5).
+         [Array, String]
+property :icmp_blocks,   # (as): array of icmp-blocks. See icmp-block tag in firewalld.policy(5).
+         [Array, String]
+property :ingress_zones, # as: array of zone names. See ingress-zone tag in firewalld.policy(5).
+         [Array, String]
+property :masquerade,    # (b): see masquerade tag in firewalld.policy(5).
+         [true, false]
+property :ports,         # (a(ss)): array of port and protocol pairs. See port tag in firewalld.policy(5).
+         [Array, String]
+property :priority,      # (i): see priority tag in firewalld.policy(5).
+         Integer
+property :protocols,     # (as): array of protocols, see protocol tag in firewalld.policy(5).
+         [Array, String]
+property :rich_rules,    # (as): array of rich-language rules. See rule tag in firewalld.policy(5).
+         [Array, String]
+property :services,      # (as): array of service names, see service tag in firewalld.policy(5).
+         [Array, String]
+property :short, # (s): see short tag in firewalld.policy(5).
+         String,
+         name_property: true
+property :source_ports,  # (a(ss)): array of port and protocol pairs. See source-port tag in firewalld.policy(5).
+         [Array, String]
+property :target,        # (s): see target attribute of policy tag in firewalld.policy(5).
+         String
+property :version,       # (s): see version attribute of policy tag in firewalld.policy(5).
+         String
```

Comment:
Instead of comments here can you add the description attribute to the property and make the comment the description. Also would help to have descriptions on all the other properties in the other resources as well as the resource documentation properties tables.

Reply:
Thanks for the hint, I did it for this file, the rest will follow. But since all information, that is needed to create the documentation, is now available in the resource: Is there any way to generate the [docs](https://github.com/a7b81a9086/firewall/blob/firewalld_custom_resource/documentation/resources/firewalld_policy.md)?
I am thinking of getting access to the resource object via the chef-library, so that I can just query all properties/default values/descriptions. Maybe there is even some cli to do just that?